### PR TITLE
Issue 6422: Add info log message.

### LINKED ISSUE #6422: Controller does not log create success for the idempotent case when the entity already exists.
**Describe the bug**
If a Reader Group already exists and a new request to create the same is received by Controller, it responds to the request with a SUCCESS status but Controller logs do not indicate that the request was successful because the Reader group already existed.

**To Reproduce**
Invoke CreateReaderGroup API on Controller and when it succeeds again invoke the same API.
The 2nd API call completes successfully (idempotent case), but in Controller logs its very hard to trace what happened to the 2nd createReaderGroup request since there is no log that indicates that Reader Group already existed.

**Additional information**
Add a new info level log message that indicates creation is sucessful as Reader Group already exists:
https://github.com/pravega/pravega/blob/master/controller/src/main/java/io/pravega/controller/task/Stream/StreamMetadataTasks.java#L378
Ditto for a Stream and a KeyValueTable

### PR DESCRIPTION
**Pull request by Bhupender-Y**
**Change log description**  
If reader group with same name already exist then printed a info log message.

**Purpose of the change**  
 Fixes #6422 

**What the code does**  
 An info log message "reader group already exist" prints, If a Reader Group already exists and a new request to create the same is received by Controller.

**How to verify it**  
Invoke CreateReaderGroup API on Controller and when it succeeds again invoke the same API. On 2nd Api call an info log message "Reader group already exist" should appears.

### CODE REVIEW

**Comment by RaulGracia** [comment]
@Bhupender-Y the DCO check is not passing, probably because you have not signed off your commits. Please, have a look to the contributing guidelines https://github.com/pravega/pravega/wiki/Contributing

**Comment by shshashwat** [comment]
@Bhupender-Y, please do the same for KVT and Streams as well.

File: controller/src/main/java/io/pravega/controller/task/Stream/StreamMetadataTasks.java [hunk 1]
```diff
@@ -375,6 +375,7 @@ private ReaderGroupConfiguration getRGConfigurationFromRecord(final String scope
                                          .thenCompose(x -> eventHelper.checkDone(() -> isRGCreated(scope, rgName, context))
                                          .thenCompose(done -> buildCreateSuccessResponse(scope, rgName, context)))));
                      }
+                     log.info("Reader Group {} already exist", rgName);
```
Comment (shshashwat) [inline controller/src/main/java/io/pravega/controller/task/Stream/StreamMetadataTasks.java:378]:
```suggestion
                     log.info("Reader Group {} already exists", rgName);
```

**Review by shshashwat — CHANGES_REQUESTED** [review]
@Bhupender-Y , please do the same for KVT and Streams as well as mentioned in the issue.

File: controller/src/main/java/io/pravega/controller/task/Stream/StreamMetadataTasks.java (hunk 1 above)
Comment (pbelgundi) [reply controller/src/main/java/io/pravega/controller/task/Stream/StreamMetadataTasks.java:378]: Please print scoped readergroup name here. You can use utility methods from NameUtils.java to build a scoped ReaderGroup name.

Comment (pbelgundi) [reply controller/src/main/java/io/pravega/controller/task/Stream/StreamMetadataTasks.java:378]: Please print scoped readergroup name here. You can use utility methods from NameUtils.java to build a scoped ReaderGroup name.

**Review by pbelgundi — CHANGES_REQUESTED** [review]

**Comment by pbelgundi** [comment]
@Bhupender-Y pls update your PR branch from master

File: controller/src/main/java/io/pravega/controller/task/Stream/StreamMetadataTasks.java (hunk 1 above)
Comment (Bhupender-Y) [reply controller/src/main/java/io/pravega/controller/task/Stream/StreamMetadataTasks.java]: Changed log message as per suggestion.

File: controller/src/main/java/io/pravega/controller/task/Stream/StreamMetadataTasks.java [hunk 2]
```diff
@@ -375,6 +375,7 @@ private ReaderGroupConfiguration getRGConfigurationFromRecord(final String scope
...
                      return buildCreateSuccessResponse(scope, rgName, context);
```
[the remark this answers is not in the document]
Comment (Bhupender-Y) [reply controller/src/main/java/io/pravega/controller/task/Stream/StreamMetadataTasks.java:379]: Added log message for createStream and createKeyValueTable APIs.

File: controller/src/main/java/io/pravega/controller/server/ControllerService.java [hunk 3]
```diff
@@ -383,6 +383,7 @@ public static long nextRequestId() {
                                   return CreateStreamStatus.newBuilder().setStatus(status).build();
                               }, executor);
                           } else {
+                              log.info("Stream {} already exists ", NameUtils.getScopedStreamName(scope, stream));
```
Comment (pbelgundi) [inline controller/src/main/java/io/pravega/controller/server/ControllerService.java:386]: Please add requestId to to the log message.

File: controller/src/main/java/io/pravega/controller/task/KeyValueTable/TableMetadataTasks.java [hunk 4]
```diff
@@ -147,6 +147,7 @@ public void initializeStreamWriters(final EventStreamClientFactory clientFactory
                                                    .thenCompose(x -> isCreateProcessed(scope, kvtName, kvtConfig,
                                                            createTimestamp, executor, context));
                                        }
+                                       log.info("KeyValue table {}/{} already exists", scope, kvtName);
```
Comment (pbelgundi) [inline controller/src/main/java/io/pravega/controller/task/KeyValueTable/TableMetadataTasks.java:150]: Add requestId

File: controller/src/main/java/io/pravega/controller/task/Stream/StreamMetadataTasks.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (pbelgundi) [reply controller/src/main/java/io/pravega/controller/task/Stream/StreamMetadataTasks.java:379]: add request id

**Review by pbelgundi — CHANGES_REQUESTED** [review]

File: controller/src/main/java/io/pravega/controller/task/Stream/StreamMetadataTasks.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (Bhupender-Y) [reply controller/src/main/java/io/pravega/controller/task/Stream/StreamMetadataTasks.java:379]: Added request id in info log message.

File: controller/src/main/java/io/pravega/controller/server/ControllerService.java (hunk 3 above)
Comment (Bhupender-Y) [reply controller/src/main/java/io/pravega/controller/server/ControllerService.java]: Added request id in info log message.

File: controller/src/main/java/io/pravega/controller/task/KeyValueTable/TableMetadataTasks.java (hunk 4 above)
Comment (Bhupender-Y) [reply controller/src/main/java/io/pravega/controller/task/KeyValueTable/TableMetadataTasks.java]: Added request id in info log message.

**Review by pbelgundi — APPROVED** [review]
LGTM

File: controller/src/main/java/io/pravega/controller/task/KeyValueTable/TableMetadataTasks.java (hunk 4 above)
Comment (Bhupender-Y) [reply controller/src/main/java/io/pravega/controller/task/KeyValueTable/TableMetadataTasks.java]: System test runs were successful: **pravega-branch-builder/1788/**

**Pull request merged**